apps/www/components/Nav/index.tsx

```diff
@@ -225,7 +226,11 @@ const Nav = () => {
                   <Link href="/" as="/">
                     <a className="block w-auto h-6">
                       <Image
-                        src={isDarkMode ? supabaseLogoWordmarkDark : supabaseLogoWordmarkLight}
+                        src={
+                          theme === 'dark' || showLaunchWeekNavMode
```

Comment:
The `showLaunchWeekNavMode` should also infer from the theme and with this implementation it seems to override some of its logic.

Basically the problem I'm seeing is that the logo should be negative also when the dropdown is open in homepage and LW pages.

<img width="1708" alt="Screenshot 2023-09-25 at 14 11 08" src="https://github.com/supabase/supabase/assets/25671831/5e8ba69b-4230-42b8-8d17-fba6d9693b10">